packages/web-components/src/components/content-switcher/content-switcher.ts

```diff
@@ -215,6 +263,18 @@ class CDSContentSwitcher extends LitElement {
   @property({ reflect: true })
   value = '';
 
+  /**
+   * Specify a selected index for the initially selected content
+   */
+  @property({ type: Number })
+  selectedIndex = 0;
+
+  /**
+   * Choose whether or not to automatically change selection on focus when left/right arrow pressed. Defaults to 'automatic'
+   */
+  @property()
```

Comment:
in web-components we try to use kebab-case when attributes are two words, so we do something like: https://github.com/carbon-design-system/carbon/pull/19896/files#diff-2345315a5d455e7d389983c3229d62a7ce38f7437b865df26624135d272e8b24R76: 
```suggestion
  @property({ attribute: 'selection-mode'})
```
then in the storybook it would go from this:
```
<cds-content-switcher
        selectionMode="${selectionMode}"
```
to 
```
<cds-content-switcher
        selection-mode="${selectionMode}"
```

... same applies for the `lowContrast` attribute a few lines below